src/imp/noop_imp.js

```diff
@@ -1,18 +1,25 @@
+function NoopSpanContextImp() {
+}
+NoopSpanContextImp.prototype.setBaggageItem = function (key, value) {
+}
+NoopSpanContextImp.prototype.getBaggageItem = function (key) {
+}
+
 function NoopSpanImp() {
 }
 NoopSpanImp.prototype.tracer = function () {
     return null;
 }
+NoopSpanImp.prototype.context = function () {
+    // XXX what's the deal here? Why can tracer() return null?
```

Comment:
The NoopSpanImp is misleading per my comment about this being used solely for unit tests.  The consumer of the `opentracing` package will never see `null` returned from any methods.  See the line I linked to in `span.js`.